CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Removed
+- App purchases and all code related to it
```

Comment:
Good catch, I'm removing it!

How did you find the unused exports and code? I tried using https://github.com/pzavolinsky/ts-unused-exports but is not clear how to get to those lines in your comments

Reply:
There's a feature in VSCode. If you right-click on an object's name and select the `Find All References` option, you'll get a list with all the appearances of that object in the repo.

<img width="844" alt="Captura de Tela 2021-05-21 às 13 49 23" src="https://user-images.githubusercontent.com/67928158/119171985-b8ba8380-ba3b-11eb-949a-e3276c410599.png">

I've only analyzed `src/api/modules/utils.ts` (since you've changed it); there might be some further unused code throughout this project. :harold-pain:

Reply:
@valentino-amadeus I think that the exports you listed are all used in other files. I deleted the unused code from `180` to `296` in 053779b6273e0f0a312f4b967c44be44ebfe4f9b , thanks!